functions.opengraph.php

```diff
@@ -261,22 +261,20 @@ function jetpack_og_tags() {
 }
 
 function jetpack_og_get_image( $width = 200, $height = 200, $max_images = 4 ) { // Facebook requires thumbnails to be a minimum of 200x200
-	$image = '';
+	$image = array();
 
 	if ( is_singular() && ! is_home() ) {
 		global $post;
```

Comment:
Would it be good to move this to the top of the function as well?  I'd always been taught that it's best to declare globals all at the beginning of the function, so they don't get lost when skimming.